### ISSUE TITLE
Weird custom attribute panick

### ISSUE THREAD

**Issue by Yatekii** [antoyo/relm#97]
Hi

With relm v0.14.0 from today, I get this weird panick under Windows.
Is this a problem of nightly rust or am I doing something wrong?

```
error: custom attribute panicked
   --> src\main.rs:108:1
    |
108 | #[widget]
    | ^^^^^^^^^
    |
    = help: message: failed to write to stdin: Os { code: 109, kind: BrokenPipe, message: "The pipe has been ended." }

error: aborting due to previous error

error: Could not compile `schema_gui`.

To learn more, run the command again with --verbose.
```

Best,
Yatekii

**Comment by antoyo** [comment]
Could you please show your code?
As I don't have Windows, it's hard to know what's wrong.

**Comment by Yatekii** [comment]
Already found it: https://github.com/antoyo/relm/blob/master/relm-attributes/src/lib.rs#L63

Would happen on linux too when rustfmt is not present ;) So maybe we should detect that.

**Comment by antoyo** [comment]
Well, we should remove the logging (which uses rustfmt) because it's not needed anymore since relm now has better error message.

**Comment by Yatekii** [comment]
Sounds plausible =) Everything else seems to be working well :)

**Comment by zzeroo** [comment]
:bowing_man: @antoyo Your wish is my order :)

**Comment by antoyo** [comment]
Fixed by #98.
